src/Controller/SpaceController.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace App\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Request;
+use App\Entity\Space;
+use App\Form\SpaceType;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+
+class SpaceController extends AbstractController
+{
+    /**
+     * @Route("/space/new", name="app_add_space")
+     */
+
+    public function new(Request $request): Response
+    {
+        $spaceForm = new Space();
+
+        $form = $this->createForm(SpaceType::class, $spaceForm);
+        $form->handleRequest($request);
+
+        if ($form->isSubmitted() && $form->isValid()) {
+            $entityManager = $this->getDoctrine()->getManager();
```

Comment:
follow good practices and use dependency injection. in fact, just use a make:crud : that will generate the right code

Reply:
done